### DISCUSSION TITLE
change instructions to allow connection to a headscale server

### DISCUSSION THREAD

**Discussion by advisorgee** [jamesog/tailscale-edgeos#31] [General]
Can the readme be updated to include the option to use a custom login server (headscale)?

tailscale up **--login-server=https://server.example.com:8080** --advertise-routes 192.0.2.0/24 --advertise-exit-node --authkey tskey-XXX

**Comment by jamesog** [comment]
There isn't any value in enumerating all possible options to `tailscale up`. People should be referring to the official Tailscale docs for how to use the CLI tools. The section that's there does say that it's an example.
